src/Tribe/Tickets_View.php

```diff
@@ -196,7 +197,13 @@ public function update_tickets() {
 
 		$post_id = get_the_ID();
 
-		$attendees = ! empty( $_POST['attendee'] ) ? $_POST['attendee'] : [];
+		$attendees = [];
+
+		if ( isset( $_POST['tribe_tickets']['attendees'] ) ) {
+			$attendees = $_POST['tribe_tickets']['attendees'];
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['tribe_tickets']['attendees']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: 230b39fadb5138a8404cc49dfb228329</sub>